### DISCUSSION TITLE
Import shortcut

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#18754] [Feature Requests]
What would be the equivalent of intellij's "Alt + Enter" for importing dependencies?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
